tests/kas_with_instance.spec.ts

```diff
@@ -177,13 +177,57 @@ test('test instance dashboard on instance name click', async ({ page }) => {
   await expect(page.getByTestId('pageKafka-tabDashboard')).toHaveCount(1);
 });
 
+// test_4kas.py test_kafka_dashboard_default
+test('test kafka dashboard default', async ({ page }) => {
+  await waitForKafkaReady(page, testInstanceName);
+  await page.locator('a', { hasText: `${testInstanceName}` }).click();
+  await expect(page.locator('h1', { hasText: `${testInstanceName}` })).toHaveCount(1);
+  await expect(page.locator('button').locator('span', { hasText: 'Dashboard' })).toHaveCount(1);
+  await expect(page.locator('h3', { hasText: 'Topics' })).toHaveCount(1);
+  await expect(page.locator('h3', { hasText: 'Topic partitions' })).toHaveCount(1);
+  await expect(page.locator('h3', { hasText: 'Consumer groups' })).toHaveCount(1);
+  await expect(page.locator('h3', { hasText: '0' })).toHaveCount(3);
+});
```

Comment:
can't we put this check to already existing test to same some time during test execution?

Reply:
If this test would stay here, perhaps it could be extended with this scenario https://issues.redhat.com/browse/MGDSTRM-8065 
- create a new topic that would use 95% of the limit
- Check for alert
- edit the topic to fill 100% of the limit
- Try to create another one and expect error

Reply:
ok I expand the existing test to cover that scenario

Reply:
@mmusil the feature to be tested on UI is still under development and would delay this merge. So I suggest to move the logic into another existing test as Jakub mentioned

Reply:
moved the logic into the 'test instance dashboard on instance name click'

Reply:
@mmusil @Frawless Can I resolve this conversation?

Reply:
From my POV yes